app/controllers/postal_addresses_controller.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class PostalAddressesController < ApplicationController
```

Comment:
Are you planning to make use of this controller later? I may overlook things, but I don't see it being used right now 👀 
If you're not using it, best get rid of it. Any unused code is code we have to maintain and results in some work 😉